frontend/src/org/components/EventForm/steps/ParticipantsStep.module.scss

```diff
@@ -55,6 +56,12 @@ h2 {
   tbody tr td {
     border-bottom: 1px solid #d9d9d9;
   }
+
+  tbody tr td:not(.oneCellRow) {
+    background-color: white;
+    white-space: nowrap;
+  }
+
```

Comment:
I think it would be great to add this somewhere into .table class:
  @include media.for-phone-only {
    max-width: inherit;
    overflow: auto;
    display: block;
  }
Scrolling right makes it do this now:
![image](https://github.com/hnuti-brontosaurus/bis/assets/33609638/16b2290b-75d0-4460-9492-136d195d863e)
But the change will allow to scroll only the table and keep the rest in place:
![image](https://github.com/hnuti-brontosaurus/bis/assets/33609638/dda16cfb-cddf-4b33-9a12-9dbf99d1fe7f)